wp-cli/class-theme-check-cli.php

```diff
@@ -0,0 +1,179 @@
+<?php
+
+if ( defined( 'WP_CLI' ) && WP_CLI ) {
+	$parent_dir = dirname( __DIR__ );
+	require_once $parent_dir . '/checkbase.php';
+	require_once $parent_dir . '/main.php';
+	WP_CLI::add_command( 'theme-check', 'Theme_Check_CLI' );
+}
+
+class Theme_Check_CLI {
+	/**
+	 * Run a theme check on the specified theme or the current theme.
+	 *
+	 * @param array $args       Indexed array of positional arguments.
+	 * @param array $assoc_args Associative array of options.
+	 * @return void
+	 *
+	 * ## OPTIONS
+	 * [<theme>]
+	 * : The slug of the theme to check. If not provided, checks the current theme.
+	 *
+	 * [--format=<format>]
+	 * : Output format. Accepts 'cli' or 'json'. Default: 'cli'.
+	 *
+	 * ## EXAMPLES
```

Comment:
Recommended to follow this format with short explanation of the command. https://github.com/wp-cli/entity-command/blob/main/src/Post_Command.php#L467

Reply:
done here: https://github.com/WordPress/theme-check/pull/458/commits/75c2f9cf123ebb6ec2272436cf421429dd6ad7d8